tools/install-deps

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+
+set -xue
+
+if [[ "$OSTYPE" =~ "linux" ]]; then
+    if which apt-get; then
+        # Debian or Ubuntu
```

Comment:
Just a little heads-up: PCLinuxOS is an RPM based distro that uses apt as its installation frontend, but the package names are still similar to other RPM distros like Fedora.